R/aaa_fabricate.R

```diff
@@ -76,6 +76,16 @@
 #'   cities = modify_level(runoff = rnorm(N))
 #' )
 #'
+#' # fabricatr can add variables that are higher-level summaries of lower-level
+#' # variables via a split-modify-combine logic and the \code{by} argument
+#'
+#' multi_level_df <-
```

Comment:
No need to create a variable `multi_level_df` here, just show the usage directly.

Reply:
It may also be worth showing an example of how magic N interacts with by.

Reply:
Good idea, happy to include if you don't mind sending on such an example.